src/helpers.c

```diff
@@ -144,18 +144,18 @@ int create_required_dirs(void)
 	int ret = 0;
 	int i;
 	char *dir;
-#define STATE_DIR_COUNT 3
+#define state_dir_COUNT 3
```

Comment:
hey, I sed'd that as well?... good catch, let me fix. That one, not
intended to be changed.

On Mon, Apr 11, 2016 at 2:13 PM, Patrick McCarty <notifications@github.com>
wrote:

> In src/helpers.c
> <https://github.com/clearlinux/swupd-client/pull/35#discussion_r59263359>:
>
> > @@ -144,18 +144,18 @@ int create_required_dirs(void)
> >  	int ret = 0;
> >  	int i;
> >  	char *dir;
> > -#define STATE_DIR_COUNT 3
> > +#define state_dir_COUNT 3
>
> Since this one is a local macro, I would prefer that it remain named
> STATE_DIR_COUNT
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/clearlinux/swupd-client/pull/35/files/4bbe7cda9873dd196d837219bef94f800ec1a1e5#r59263359>
>